docs/admins.rst

```diff
@@ -113,6 +119,53 @@ the x86_64 platform will publish v1 images as well as v2 images. Other
 platforms on instance1, and all platforms on instance2, will only
 publish v2 images.
 
+Autorebuilds and OSBS updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OSBS’s autorebuild feature automatically starts new builds of layered images
+whenever the base parent image changes. This has implications for supporting
+these builds across OSBS updates.
+
+A code release of OSBS tooling means changing the build image (AKA "buildroot",
+specified as ``build_from`` in ``osbs.conf``) to be used for building
+containers.
+
+The build image used for building container images is defined
+in the ``Build``/``BuildConfig`` OpenShift objects under
+``.spec.strategy.customStrategy.from`` object. This can be a full reference to
+a specific container image in a container registry; or it can reference an
+ImageStreamTag object.
+
+Any explicit container image build will put the newer build image as configured
+into the ``BuildConfig`` object. But for autorebuilds, the ``BuildConfig``
+objects are not updated; when OpenShift detects a change in parent image and
+starts a new build for the layered image, it will use the outdated build image.
+
+Updating ``BuildConfig`` objects after OSBS updates
+'''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To work around this limitation, image owners could start explicit builds for
+each image in the hierarchy of images to update the underlying ``BuildConfig``
+object to use the newer build image. But this is a tedious and error-prone
+approach.
+
+Latest Released Build Image
+'''''''''''''''''''''''''''
+
+To avoid having to update each ``BuildConfig`` object to use a newer build
+image, the build image should be specified with a transient tag. Transient
+tags are those that are meant to reference different images over time. To use a
+newer build image, simply move the transient tag to reference it. For instance,
```

Comment:
@lcarva @rcerven is this accurate? Do we actually do this? I wasn't sure if OpenShift would check that the tag has been updated and now points at a new buildroot.

Reply:
I don't think OpenShift will do that. There's a way to set the policy so it does check for updates, but I think that might be available just for pods, not builds.

Reply:
Are we currently just letting autorebuilds use old buildroots?

The transient tag approach *would* work, if just combined with removing the previous tagged version from docker after a code update...

But for us specifically I imagine we want to record exactly which buildroot was used.